bundle/src/main/java/com/adobe/acs/commons/fam/package-info.java

```diff
@@ -17,5 +17,5 @@
  * limitations under the License.
  * #L%
  */
-@aQute.bnd.annotation.Version("2.2.0")
+@aQute.bnd.annotation.Version("3.0.0")
```

Comment:
It is hard to tell, but I'm guessing that this is due to `ThrottledTaskManager` missing a `@ProviderType` annotation.